src/doc/rustdoc/src/write-documentation/re-exports.md

```diff
@@ -0,0 +1,128 @@
+# Re-exports
+
+Let's start by explaining what are re-exports. To do so, we will use an example where we are
```

Comment:
I don't have time to do this myself but would someone be able to proofread this for grammar?